src/ParticleSpecies.cpp

```diff
@@ -116,6 +118,31 @@ namespace
     }
 }
 
+void
+ParticleSpecies::seriesFlush()
```

Comment:
@franzpoeschel Do you have an idea how we can implement this method safely here? :)

Reply:
[Little hint](https://github.com/openPMD/openPMD-api/blob/242cb1dbf5c508f6f8da3669f48fe26bbfc35b37/src/backend/Attributable.cpp#L151) ;) 
Needed to do the same trick for #901

Reply:
Ah perfect, I was close. Will try that construct, let's see :)

Reply:
That works! :) Thank you :sparkles: